src/components/PopupWithForm.js

```diff
@@ -8,26 +8,28 @@ export default class PopupWithForm extends Popup {
       this._popupForm.querySelectorAll(".modal__input")
     );
     this._handleFormSubmit = handleFormSubmit;
+   
   }
 
```

Comment:
**COULD BE IMPROVED**

You can make a special method for getting a form element from `PopupWithForm` not to **search** it in `index.js`



```js

  getForm() {
    return this._popupForm
  }


```